src/test/java/org/opensearch/geospatial/rest/action/upload/geojson/RestUploadGeoJSONActionIT.java

```diff
@@ -54,7 +57,8 @@ public void testGeoJSONUploadFailIndexExists() throws IOException {
 
     public void testGeoJSONUploadSuccessPutMethod() throws IOException {
 
-        String index = randomLowerCaseString();
+        String index = randomLowerCaseStringWithSuffix(ACCEPTED_INDEX_SUFFIX_PATH);
+        ;
```

Comment:
same, we don't need this single ";" here

Reply:
Ack